MATLAB/Utilities/filtering.m

```diff
@@ -43,9 +44,11 @@
     
     fproj = (real(ifft(fproj)));
     
-  
-    proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  size(angles,2)   )/2*(geo.DSD(ii)/geo.DSO(ii));
-    
+    if parker
+   	proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  (pi/angle_step)   )/2*(geo.DSD(ii)/geo.DSO(ii));

```

Comment:
Hi! Could you explain this line please? I think I understand why you are doing this, but would it not make sense to get rid of the `2*pi` somehow? Can you share some maths/paper for me to understand this?

Reply:
Hi Ander, I referred to code by Kyungsang Kim (https://www.mathworks.com/matlabcentral/fileexchange/35548-3d-cone-beam-ct-cbct-projection-backprojection-fdk-iterative-reconstruction-matlab-examples), and do not fully understand the math yet. I will try to figure it out later. 

BTW, which reference did you use for the filtering weighting for non-Parker one? 
proj(:,:,ii) = fproj(end/2-geo.nDetector(1)/2+1:end/2+geo.nDetector(1)/2,:)/2/geo.dDetector(1)*(2*pi/  size(angles,2)   )/2*(geo.DSD(ii)/geo.DSO(ii));

Reply:
I actually took it from there! Ah, I removed the parker apparently  (this was over 6 years ago).
My bad!